### ISSUE TITLE
Python 2.6.9 on OS X fails because of missing `argparse`

### ISSUE THREAD

**Issue by aaaaalbert** [SeattleTestbed/continuous-integration#7]
This is really not a pressing issue, but I wonder why the same Travis config [worked a bit ago](https://travis-ci.org/aaaaalbert/repy_v2/jobs/255832093), but now [fails and complains](https://travis-ci.org/aaaaalbert/repy_v2/jobs/274098655#L93) about missing the `argparse` module now:

```
$ $PythonBin utf.py -a;
Traceback (most recent call last):
  File "utf.py", line 54, in <module>
    import argparse
ImportError: No module named argparse
```

We had an issue with `argparse` and an old Python version (2.5) previously, see fba8bf3475e4615d049de916456fdf9cd039fece, but this time it seems that things have changed on Travis' side.

Anyway, I'd like to emphasize that testing on a very old Python version is not our primary focus.

**Comment by lukpueh** [comment]
Might be related to a recent Travis update after which our py26 unit tests over in securesystemslib didn't work anymore (cf. https://github.com/secure-systems-lab/securesystemslib/pull/58)?

**Comment by aaaaalbert** [comment]
Good catch! The solution here will probably follow what @securesystemslib did --- remove Python 2.6 testing.
